pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroup.java

```diff
@@ -78,6 +80,9 @@ protected ResourceGroup(ResourceGroupService rgs, String name,
         this.setResourceGroupMonitoringClassFields();
         this.setResourceGroupConfigParameters(rgConfig);
         this.setDefaultResourceUsageTransportHandlers();
+        this.resourceGroupPublishLimiter = new ResourceGroupPublishLimiter(rgConfig, rgs.getPulsar().getExecutor());
```

Comment:
The default thread pool(pulsar executor) size is 20. However, as the ResourceGroup number increases it gives pressure on the thread pool to process.
Is it better to provide a new thread pool here to process? It means one RG has one thread pool.

Reply:
Another question:
There is only `PublishLimiter`. It seems we need another limiter called `DispatchLimiter`. right?